cloudinit/apport.py

```diff
@@ -34,12 +34,14 @@
     'OpenStack',
     'Oracle',
     'OVF',
+    'RbxCloud (HyperOne, Rootbox, Rubikon)',
```

Comment:
Added RbxCloud according guides how to create datasource.

Reply:
There are a couple of other entries here that have similar formatting:

```
    'GCE - Google Compute Engine',
    'IBM - (aka SoftLayer or BlueMix)',
```

Should this perhaps be 
```suggestion
    'RbxCloud - (HyperOne, Rootbox, Rubikon)',
```

for consistency?